### PR TITLE
Zamiatina

### PR DESCRIPTION
**Pull request by Katerina-Zamiatina**
Update modal resume to common portal

### CODE REVIEW

File: src/components/Library/LibraryBook/LibraryBook.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const LibraryBook: React.FC<BookProps> = ({ book }) => {
           <span className={styles.subtitle}>{book.pages}</span>
         </div>
         {book.status === 'done' ? (
```
Comment (MarinaTripetska) [inline src/components/Library/LibraryBook/LibraryBook.tsx:35]: maybe for 'done' and another status create some enum?

File: src/locales/en/translation.json [hunk 2]
```diff
@@ -121,7 +121,7 @@
       "name": "Kateryna",
       "role": "Developer",
       "gitHubPath": "https://github.com/Katerina-Zamiatina",
-      "cases": ["do this", "and this", "and this this"]
+      "cases": ["user authorisation", "login/register page", "library page"]
```
Comment (MarinaTripetska) [inline src/locales/en/translation.json:124]: You can add more - +2-3

File: src/services/books/books-service.ts [hunk 3]
```diff
@@ -57,13 +60,19 @@ const getBooksByStatus = async (status: string) => {
 };
 
 const createBook = async ({ name, author, year, pages }: BookT) => {
```
Comment (MarinaTripetska) [inline src/services/books/books-service.ts:62]: you don't need de structurization here

File: src/components/Library/LibraryBook/LibraryBook.tsx (hunk 1 above)
Comment (Katerina-Zamiatina) [reply src/components/Library/LibraryBook/LibraryBook.tsx:35]: Ok, good idea

File: src/locales/en/translation.json (hunk 2 above)
Comment (Katerina-Zamiatina) [reply src/locales/en/translation.json:124]: I will think, but in short that's all i've done)

File: src/services/books/books-service.ts (hunk 3 above)
Comment (Katerina-Zamiatina) [reply src/services/books/books-service.ts:62]: you mean destructure? Yes, will change this one

File: src/locales/en/translation.json (hunk 2 above)
Comment (MarinaTripetska) [reply src/locales/en/translation.json:124]: ок:)

File: src/services/books/books-service.ts (hunk 3 above)
Comment (MarinaTripetska) [reply src/services/books/books-service.ts:62]: yes, sorry, I'm not good at English :)

**Review by MarinaTripetska — APPROVED** [review]

**Pull request merged**